controller/readonly.go

```diff
@@ -34,18 +32,8 @@ func NewReadOnly(conf *config.Config) (Controller, error) {
 }
 
 // Init initializes the controller before it can be run
-func (ctrl *ReadOnly) Init(ctx context.Context) (*driver.Drivers, error) {
-	drivers, err := ctrl.init(ctx)
-	if err != nil {
-		return nil, err
-	}
-
-	// Sort units for consistent ordering when inspecting tasks
-	sort.Slice(ctrl.units, func(i, j int) bool {
-		return ctrl.units[i].taskName < ctrl.units[j].taskName
-	})
```

Comment:
It looks like the inspect task sorting might be removed (vs. refactored). Just wanted to double-check if that was intentional?

Reply:
sorting is removed since `units` are no longer needed, `driver.Drivers` is a map. This probably would have been better with PR https://github.com/hashicorp/consul-terraform-sync/pull/339

Reply:
Ah, no worries. I can see it easier to remove here when splitting apart changes. Makes sense why it's removed!